lib/pwnlib/tubes/tube.rb

```diff
@@ -159,7 +166,7 @@ def recvuntil(delims, drop: false, timeout: nil)
             while @timer.active?
               s = recv(1)
 
-              return '' if s.empty?
```

Comment:
Is this change needed?

Reply:
No. I raised it in issue #91, and it was classified as "notbug".

Reply:
So please rollback it